compiler-rt/test/tsan/pthread_atfork_deadlock3.c

```diff
@@ -28,17 +28,17 @@ void *worker(void *main) {
 }
 
 void atfork() {
+  write(2, "in atfork\n", strlen("in atfork\n"));
   barrier_wait(&barrier);
   barrier_wait(&barrier);
-  write(2, "in atfork\n", strlen("in atfork\n"));
```

Comment:
..because write now triggers the signal handler, causing the two strings to come out in opposite order